packages/codegen-ui-react/lib/__tests__/studio-ui-json/componentWithUserSpecificAttributes.json

```diff
@@ -2,57 +2,35 @@
   "id": "1234-5678-9010",
   "componentType": "Flex",
   "name": "Profile",
-  "bindingProperties": {
-    "username": {
-      "type": "Authentication",
-      "bindingProperties": {
-        "userAttribute": "username"
-      }
-    },
-    "userImage": {
-      "type": "Authentication",
-      "bindingProperties": {
-        "userAttribute": "picture"
-      }
-    },
-    "customUserAttributeIcecream": {
-      "type": "Authentication",
-      "bindingProperties": {
-        "userAttribute": "custom:favorite_icecream"
-      }
-    }
-  },
   "children": [
     {
+      "name": "child1",
       "componentType": "Image",
       "properties": {
         "src": {
-          "bindingProperties": {
-            "property": "userImage"
-          }
+          "userAttribute": "username"
         }
       }
     },
     {
+      "name": "child2",
       "componentType": "Button",
       "properties": {
         "label": {
-          "bindingProperties": {
-            "property": "username"
-          }
+          "userAttribute": "picture"
         }
       }
     },
     {
+      "name": "child3",
       "componentType": "Button",
       "properties": {
         "label": {
-          "bindingProperties": {
-            "property": "customUserAttributeIcecream"
-          }
+          "userAttribute": "customUserAttributeIcecream"
```

Comment:
Could you leave this as custom:favorite_icecream (this is the actual format which a custom attribute takes from cognito.

Reply:
Done